ingest/build-configs/ncbi/defaults/annotations.tsv

```diff
@@ -0,0 +1,9 @@
+# Manually curated annotations TSV file
+# The TSV should not have a header and should have exactly three columns:
+# id to match existing metadata, field name, and field value
+# If there are multiple annotations for the same id and field, then the last value is used
+# Lines starting with '#' are treated as comments
+# Any '#' after the field value are treated as comments.
+A/environment/USA/CO-UW-9084466/2024	division	Colorado	# Based on linked BioSample https://www.ncbi.nlm.nih.gov/biosample/SAMN41462555
```

Comment:
Entries in `annotations.tsv` are not currently being applied to the fetched and parsed metadata. If I inspect `ingest/ncbi/results/metadata.tsv` I get division=USA for `A/environment/USA/CO-UW-9084466/2024`.

Reply:
Oh, good catch. Will debug 🤔

Reply:
Fixed in https://github.com/nextstrain/avian-flu/pull/40/commits/f04d275d87472ff603b7cc7a646407a193bd541d.

I forgot that the format of the annotations.tsv only allows 2 tabs, so final comments need to be separated by a space.

Reply:
Ah! Nice catch. Thanks for fixing.